Makefile

```
@@ -0,0 +1,28 @@
+# Makefile for paper (figures and data)
+
+.PHONY: all clean
+
+all: serial_interval australia_data hawaii_data new_zealand_data ontario_data
```

```suggestion
all: serial_interval data

data: australia_data hawaii_data new_zealand_data ontario_data
```
Since serial interval is slow, this could be a useful feature. I'm just guessing on the syntax, but it seems to work.